README.md

```diff
@@ -1,2 +1,49 @@
 # Go App
-Go App is an app written in Go
+
+Go App is an app written in Go.
+This repository is used as a template.
```

Comment:
It's better to leave this out, so we don't need to bother removing it once we use this template

Reply:
Actually, updating 1 line and 2 consecutive lines looks the same to me. We still need to update it to a proper Title and Description of the project.

Reply:
actually, this template is meant to be used by CLI tool here: https://github.com/kurio/boilerplater (go project is supported but isn't documented yet), where the title & description is inputted there, and placed to the README and the rest of the project automatically.